dbgpt/storage/knowledge_graph/community/tugraph_store_adapter.py

```diff
@@ -552,10 +558,8 @@ def explore(
         if not subs:
             return MemoryGraph()
 
-        if depth is None or depth < 0 or depth > self.MAX_HIERARCHY_LEVEL:
-            # TODO: to be discussed, be none or MAX_HIERARCHY_LEVEL
-            # depth_string = ".."
-            depth = self.MAX_HIERARCHY_LEVEL
+        if depth < 0 or depth > 3:
```

Comment:
Do not impose mandatory restrictions here so that upper-level callers can modify the depth.

Reply:
fixed